openfoam-interface/internal-interface/wrapper/simphonyInterface.C

```diff
@@ -945,6 +943,9 @@ void foam_modifyNumerics(std::string name, std::string fvSch, std::string fvSol,
         cDIS()
     );
 
+    TimeMod* t = (TimeMod*)(runTimes[name]);
```

Comment:
TimeMod is a class which inheritance from the OpenFoam class Time adding a new method update(). This new method allows to modify run settings without needing a controlDict file. It is true that subclass TimeMod is never instantiated directly, but thanks to C++ polimorphism I can use its extended behavior. 
Another option was reimplementing in simphony the class Time (as was done with fvSchemes and solution classes), but this involves repeat a lot of files without almost any modification. I think this approach of inheritance is the more appropiate and clear in this case.